### PR TITLE
Changes to improve the grammer, clarity, etc., especially to cli-usage.md.  

### PR DESCRIPTION
**Pull request by cjyabraham**

### CODE REVIEW

File: docs/cli-usage.md [hunk 1]
```diff
@@ -335,7 +336,7 @@ The easiets way to interact with your DAO is using [Metamask](https://metamask.i
 
 - It's unlocked
 - Private network (_Localhost 8545_) is chosen
-- The first account provided by `aragon run` or `aragon devchain` is imported and selected. To import the account, copy the private key (something like `a8a54b2d8197bc0b19bb8a084031be71835580a01e70a45a13babd16c9bc1563`), go to the accounts upper icon, to the left of the hamburguer button, scroll down, click on `Import account` and paste the value you copied.
+- The first account provided by `aragon run` or `aragon devchain` is imported and selected. To import the account, copy the private key (something like `a8a54b2d8197bc0b19bb8a084031be71835580a01e70a45a13babd16c9bc1563`), go to the Metamask accounts upper icon (to the left of the hamburguer button), scroll down, click on "Import account" and paste the value you copied.
```
Comment (luisivan) [inline docs/cli-usage.md:339]: Should we explicitly mention MetaMask here? Feels like we should be agnostic

**Comment by luisivan** [comment]
Great changes! Left a comments. Everything else is great

File: docs/cli-usage.md (hunk 1 above)
Comment (cjyabraham) [reply docs/cli-usage.md:339]: My above change occurs in a larger section called [Using Metamask](https://hack.aragon.org/docs/cli-usage.html#using-metamask).  We also promote Metamask [in the tutorial](https://hack.aragon.org/docs/tutorial.html#metamask).  If we don't want to promote Metamask explicitly, would you like me to remove those sections entirely?  I'm concerned that then the average dev will be missing an important piece of the puzzle to get things working?

Comment (luisivan) [reply docs/cli-usage.md:339]: Yeah, fair enough. I think for now we can leave MetaMask as a must-have, but in the future we could leave it more agnostic when [Frame](https://frame.sh/) is ready for prime time

**Review by luisivan — APPROVED** [review]

File: docs/cli-usage.md (hunk 1 above)
Comment (cjyabraham) [reply docs/cli-usage.md:339]: ok, sounds good.

**Pull request merged**